target/linux/imx6/image/Makefile

```diff
@@ -206,7 +206,10 @@ define Device/cubox-i
   UBOOT := mx6cuboxi
   BOOT_SCRIPT = bootscript-cubox
   DEVICE_NAME := cubox
-  DEVICE_PACKAGES := kmod-drm-imx kmod-drm-imx-hdmi kmod-usb-hid
+  DEVICE_PACKAGES := \
+	kmod-drm-imx kmod-drm-imx-hdmi \
+	kmod-brcmfmac brcmfmac-firmware-4329-sdio brcmfmac-firmware-4329-sdio-cubox-i brcmfmac-firmware-4330-sdio brcmfmac-firmware-4330-sdio-cubox-i \
+	kmod-wl18xx wl18xx-firmware wl18xx-firmware-cubox-i
```

Comment:
Consider wrapping these lines a little more systematically, i.e. just before 80 characters for all lines or similar.

Reply:
well, I was tryi8ng to be systematic by having line by feature more or less
- display
- Broadcom wifi
- Ti wifi
However I will think about reorganizing. Any ideas?

Reply:
Well, one line is longer than allowed anyway, either wrap each line when full so no line is longer than 74 chars or just have one package per line.